swagger.yaml

```diff
@@ -6201,40 +6201,6 @@ definitions:
     type: object
     x-go-custom-tag: gorm:"embedded;embeddedPrefix:vsphere_"
     description: Vsphere platform specific configuration upon which to perform the installation
-    properties:
```

Comment:
Why not remove all of vsphere_platform, including the reference from "platform"?